CONTRIBUTING.md

```diff
@@ -172,9 +172,20 @@ contributors.
 
 There are separate directories for `stdlib` and `third_party` stubs.
 Within those, there are separate directories for different versions of
-Python the stubs target.  If the given library works on both Python 2
-and Python 3, prefer to put your stubs in an `2and3` directory, unless
-the types are so different that the stubs become unreadable that way.
+Python the stubs target.
+
+As a convention, the directory name indicates the lowest major version of Python
```

Comment:
Is "lowest major version" unclear? (Perhaps so, as it was not clear enough for you). 
I could add an 'e.g. Python 2 or Python 3' if that doesn't seem redundant.